core/rfb.js

```diff
@@ -1271,6 +1462,42 @@
                 msg_type = this._sock.rQshift8();
             }
 
+            if (this._rfb_atenikvm) {
+                switch (msg_type) {
```

Comment:
Yes, some do conflict; for example, per https://github.com/kelleyk/rfbproto/blob/master/rfbproto.rst#server-to-client-messages, 4 should be a "ResizeFrameBuffer" event.  Regardless, they're proprietary ATEN extensions, and it seems like a good idea to keep them a little bit separate.